controllers/storagecluster/noobaa_system_reconciler.go

```diff
@@ -180,9 +180,9 @@ func (r *StorageClusterReconciler) setNooBaaDesiredState(nb *nbv1.NooBaa, sc *oc
 	}
 
 	// Add KMS details to Noobaa spec, only if
-	// cluster-wide encryption is enabled (ie; sc.Spec.Encryption.Enable is True)
+	// cluster-wide encryption is enabled (ie; sc.Spec.Encryption.ClusterWide is True)
 	// and KMS ConfigMap is available
-	if sc.Spec.Encryption.Enable {
+	if sc.Spec.Encryption.ClusterWide {
```

Comment:
no need to check kms is enabled?

Reply:
`getKMSConfigMap` will return `nil, nil` if it is not enabled.

Reply:
we can avoid one network call :P

Reply:
and there is a change if someone enabling encryption for nooba without actually enabling from storagecluster

Reply:
Yup, but, it will return `nil` before polling for the resource. It will only poll if KMS is enabled.
https://github.com/red-hat-storage/ocs-operator/blob/1f34b667278a17aa9e82b6128144d50f796696ec/controllers/storagecluster/kms_resources.go#L115

Reply:
`and there is a change if someone enables KMS encryption for nooba without actually enabling it from storagecluster` -->
sorry I did not get it, how will this cause an issue ?
I mean, if they are not using storagecluster CR then they should add it to the noobaa CR directly ?? There will be no extra network call.

Reply:
ack